### ISSUE TITLE
 Anti-bactericidal/permeability-increasing protein (BPI) antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6598]
**Preferred term label:**  Anti-bactericidal/permeability-increasing protein (BPI) antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Perinuclear antineutrophil antibody positivity HP:0032229

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  I think this is c-ANCA rather than p-ANCA, see e.g. https://pubmed.ncbi.nlm.nih.gov/8697620/ and https://www.liebertpub.com/doi/10.1089/ped.2018.0970
I have placed it there -- please reopen if this is incorrect
